src/commands/dao_cmds/change-controller.js

```diff
@@ -0,0 +1,82 @@
+const TaskList = require('listr')
+const { ensureWeb3 } = require('../../helpers/web3-fallback')
+const { getContract } = require('../../util')
+const listrOpts = require('../../helpers/listr-options')
+const chalk = require('chalk')
+
+exports.command = 'change-controller <token-address> <new-controller>'
```

Comment:
Even more convinced now that these commands should be under a `dao token` or `dao minime` sub-command.